pypesto/profile/profile.py

```diff
@@ -122,18 +134,31 @@ def profile(
     profile_options = ProfileOptions.assert_instance(profile_options)
 
     # profile startpoint method
-    if create_profile_startpoint is None:
-        def create_next_startpoint(x, par_index, par_direction):
-            return fixed_step(x, par_index, par_direction,
-                              step_size=profile_options.default_step_size)
+    if next_guess_method is None:
+        def create_next_guess(x, par_index, par_direction, profile_options,
+                              current_profile, problem, global_opt):
+            return next_guess(x, par_index, par_direction, profile_options,
+                              'adaptive_step_regression', current_profile,
+                              problem, global_opt)
+    elif isinstance(next_guess_method, str):
```

Comment:
the two cases ``None`` and ``isinstance(next_guess_method, str)`` seem pretty alike. Maybe one can do beforehand sth like
```python
if next_guess_method is None:
    # default method
    next_guess_method = 'adaptive_step_regression'
```
and then test only for str here?